### DISCUSSION TITLE
How to render skybox with cubemap whoes format is png or jpg

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#7114] [Q&A]
Hi, i am developing a vr app with filament. I want to partially replace one face of cubemap during runtime, and  png or jpg will be a suitable chioce. Howevere `filament` seems to only support render skybox with `ktx` or `rgb32f` format, how can i render skybox with cubemap whose format is png or jpg?

**Comment by romainguy** [comment]
The skybox can be almost any format you want. It's just the helper loading functions that look for HDR data. Using the Skybox API directly you can use any texture format you want.
